lib/Common/Core/CmdParser.cpp

```diff
@@ -68,11 +68,12 @@ CmdLineArgsParser::ParseString(__inout_ecount(ceBuffer) LPWSTR buffer, size_t ce
             case ' ':
             case ',':
             case 0:
-                fDone = 1;
+                fDone = true;
                 break;
-            case '-':
+            case '-': // REVIEW is this correct? What syntax results in this path being taken?
+            case '=':
```

Comment:
Is there any potential negative to adding this as a separator? I don't believe we have a special meaning for `=` in any of our arguments syntax.